src/NodaTime/DateInterval.cs

```diff
@@ -214,6 +214,31 @@ public DateInterval Intersection([NotNull]DateInterval interval)
                 : null;
         }
 
+        /// <summary>
+        /// Returns the union between the given interval and this interval, as long as they're overlapping or contiguous.
+        /// </summary>
+        /// <param name="interval">
+        /// The specified interval to unite with this one. 
```

Comment:
Not sure whether this is a little ambiguous - it sounds like it *might* mutate this one. I'm struggling to find better wording though...

(As an aside, we generally collapse simple tags like this onto a single line, although it doesn't matter.)